index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <title>Minesweep</title>
+    <link rel="stylesheet" href="style.css" />
+    <script type="module" src="script.js"></script>
+</head>
+
+<body>
+    <header>
+        <h1>Minesweep</h1>
```

Comment:
nitpick: the game is called minesweeper
```suggestion
    <title>Minesweeper</title>
    <link rel="stylesheet" href="style.css" />
    <script type="module" src="script.js"></script>
</head>

<body>
    <header>
        <h1>Minesweeper</h1>
```

Reply:
ok